tests/unit/phpunit/SuitePHPUnit_Framework_TestCase.php

```diff
@@ -52,7 +52,7 @@ public function setUp()
 
         global $current_user, $sugar_config;
         try {
-            $current_user = @\BeanFactory::getBean('Users'); //new User();
+            $current_user = @\BeanFactory::getBean('Users'); //\BeanFactory::newBean('Users');
```

Comment:
I don't know if we should change the comment here?

Reply:
The comments were changed by the script, my opinion would be that they should be removed but I’ll leave that to the Product Team to decide. I’ll leave them as is for now as I think it reflects the change, if the comment is meant to be there it reflects the same expression but matches its current state.